src/zenml/zen_server/zen_server.py

```diff
@@ -121,7 +124,16 @@ def __init__(
         super().__init__(config=config, **attrs)
 
     def run(self) -> None:
-        if self.config.store_profile_configuration.store_type == StoreType.REST:
+        profile = GlobalConfiguration().get_profile(self.config.profile_name)
+        print(
```

Comment:
this should be `logger.info(...)` I think

Reply:
I had a long unit-test debugging session and this must've gotten away from me. I'll remove it. Thank you for spotting it !